fe/fe-core/src/main/java/org/apache/doris/analysis/DefaultValueExprDef.java

```diff
@@ -23,28 +23,44 @@
 import org.apache.doris.catalog.Type;
 import org.apache.doris.common.AnalysisException;
 
+import com.google.common.collect.Lists;
 import com.google.gson.annotations.SerializedName;
 import org.apache.log4j.LogManager;
 import org.apache.log4j.Logger;
 
+import java.util.List;
+
 /**
  * This def used for column which defaultValue is an expression.
  */
 public class DefaultValueExprDef {
     private static final Logger LOG = LogManager.getLogger(DefaultValueExprDef.class);
     @SerializedName("exprName")
     private String exprName;
+    @SerializedName("precision")
+    private Long precision;
```

Comment:
For newly added field, better implement `GsonPostProcessable` and give it a init value